pom.xml

```diff
@@ -98,12 +98,14 @@ under the License.
   </contributors>
 
   <properties>
-    <bootstrap.version>2.3.2</bootstrap.version>
+    <bootstrap.version>3.1.1</bootstrap.version>
```

Comment:
Correct and this is wrong...

Reply:
Yes I've deciphered from one of the comments that to test this I need to run the run-its profile and then I can look at various configurations. the site.xml is for some reason referencing the previous release rather than the current version, which hid various changes.

I believe the latest update to the branch should resolve the big issues.